documentation/opencast-installation.md

```diff
@@ -23,12 +23,18 @@ In this manual we use `<annotationtool-dir>` for the base dir of the Annotation
 
     git clone https://github.com/opencast/annotation-tool.git
 
+#### Prequisites
+
+Make sure that you have openjdk-8-jdk and maven installed.
+
 ### Building the Annotation Tool
 
-    mvn clean install -DdeployTo=<opencast-dir> -Dopencast.version=<your Opencast version number>
+    mvn clean install -DdeployTo=<opencast-dir>
 
 This should build the frontend, include it into the Opencast modules and copies the JARs
-to your Opencast installation.
+to your Opencast installation. You should provide the built Opencast directory, e.g. `-DdeployTo=/home/user/opencast/build/opencast-dist-develop-8-SNAPSHOT/`
+
+There is an additional, special property `opencast.version` to specify the Opencast version number.
```

Comment:
I would leave this out. At this point, no one should have to specify this anymore. :thinking: